app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/TaskListController.scala

```diff
@@ -57,8 +53,6 @@ class TaskListController @Inject() (
     }
 
   private def startLink(implicit request: JourneyRequest[AnyContent]) =
```

Comment:
do we need this def anymore too?

Reply:
this method, it was only in place to facilitate the feature flag, can just move what ever is calling it, to just call the route

Reply:
done